frontend/src/lib/services/Libre311/Libre311.ts

```diff
@@ -265,6 +265,34 @@ export type ServiceRequest = z.infer<typeof ServiceRequestSchema>;
 export const GetServiceRequestsResponseSchema = z.array(ServiceRequestSchema);
 export type GetServiceRequestsResponse = z.infer<typeof GetServiceRequestsResponseSchema>;
 
+// ***************** Create Service *************** //
+
+// Create Service - Request Schema
+export const CreateServiceRequestSchema = z.object({
+	service_code: z.string(),
+	service_name: z.string(),
+	description: z.string(),
+	service_definition: z.string(),
+	group_id: z.number()
+});
+
+//  Create Service - Response Schema
+export const CreateServiceResponseSchema = z
+	.object({
+		id: z.number(),
+		jurisdiction_id: z.string(),
+		group_id: z.number()
+	})
+	.merge(ServiceSchema);
+
+//  Create Service - Request Type
+export type CreateServiceRequest = z.infer<typeof CreateServiceRequestSchema>;
+
+// Create Service - Response Type
+export type CreateServiceResponse = z.infer<typeof CreateServiceResponseSchema>;
+
+// ************************************************ //
```

Comment:
Hopefully our naming conventions do not get confusing but my understanding is as follows

Create Service Request - means the "Request" used to create a new "Service"
Create Service Request Request - would mean the "Request" used to create a new "Service Request"